### ISSUE TITLE
Background and foreground colors have a sufficient contrast ratio

### ISSUE THREAD

**Issue by alij2016** [GoogleChrome/web.dev#5059]
**Describe the bug**

Hi, I am pretty sure there is a bug regarding Background and foreground colors have a sufficient contrast ratio. My website, letsdiscovermauritius.com, has a black foreground and white background. After analyzing the contrast ratio, it is giving me 21, which is very good and valid. However, your system is detecting it as error. I see this on all pages with white background and text is black.

**Comment by robdodson** [comment]
It looks like Lighthouse is throwing an error on the page. Under the hood, web.dev is using the PageSpeed Insights API which is based on Lighthouse. Transferring the issue to the LH repo to see if they can help.

![image](https://user-images.githubusercontent.com/1066253/114235578-3c5d5c80-9935-11eb-9475-65fa6944d0d9.png)
